setup.py

```diff
@@ -1,18 +1,16 @@
-from distutils.core import setup
-
-#try:
-#    from vpnotebook import cmdclass
-#except:
-#    import pip, importlib
-#    pip.main(['install', 'vpython']); cmdclass = importlib.import_module('vpnotebook').cmdclass
+from __future__ import print_function
+try:
+    from setuptools import setup
+except:
+    print("no setuptools, using distutils")
```

Comment:
Might it be better to include [ez_setup.py](http://pythonhosted.org/setuptools/setuptools.html#using-setuptools-without-bundling-it) in the repo and use that to bootstrap a working `setuptools`?